srcpkgs/gdb/patches/gdb-mips-linux-nat.patch

```diff
@@ -0,0 +1,11 @@
+--- a/gdb/mips-linux-nat.c
++++ b/gdb/mips-linux-nat.c
```

Comment:
For consistency, I'd make the other packages use the same patch format.

Not really a requirement, though, mostly a suggestion.

Reply:
Fixed.